api/v1/auxilliary.yaml

```diff
@@ -0,0 +1,59 @@
+---
+openapi: 3.0.2
+info:
+  title: DSSAuxService
+  version: 0.8.0
```

Comment:
This API isn't tied to remote ID API (right?), so it should probably be versioned independently; 0.1.0 or 1.0.0 maybe?

Reply:
done